example/src/screens/menu.js

```diff
@@ -1,31 +1,40 @@
 import React, {Component} from 'react';
+
 import {Platform, StyleSheet, View, TouchableOpacity, Text, Image} from 'react-native';
 import {Navigation} from 'react-native-navigation';
 
-
 const appIcon = require('../img/app-icon-120x120.png');
 
 export default class MenuScreen extends Component {
-
   render() {
     return (
+
       <View style={styles.container}>
         <Image source={appIcon} style={styles.image}/>
         <TouchableOpacity style={styles.menu} onPress={this.onCalendarsPress.bind(this)}>
           <Text style={styles.menuText}>Calendars</Text>
         </TouchableOpacity>
-        <TouchableOpacity style={styles.menu} onPress={this.onCalendarListPress.bind(this)}>
+        <TouchableOpacity
+          style={styles.menu}
+          onPress={this.onCalendarListPress.bind(this)}>
           <Text style={styles.menuText}>Calendar List</Text>
         </TouchableOpacity>
-        <TouchableOpacity style={styles.menu} onPress={this.onHorizontalCalendarListPress.bind(this)}>
+        <TouchableOpacity
+          style={styles.menu}
+          onPress={this.onHorizontalCalendarListPress.bind(this)}>
           <Text style={styles.menuText}>Horizontal Calendar List</Text>
         </TouchableOpacity>
-        <TouchableOpacity style={styles.menu} onPress={this.onAgendaPress.bind(this)}>
+        <TouchableOpacity
+          style={styles.menu}
+          onPress={this.onAgendaPress.bind(this)}>
           <Text style={styles.menuText}>Agenda</Text>
         </TouchableOpacity>
-        <TouchableOpacity style={styles.menu} onPress={this.onExpandablePress.bind(this)}>
+        <TouchableOpacity
+          style={styles.menu}
+          onPress={this.onExpandablePress.bind(this)}>
           <Text style={styles.menuText}>Expandable Calendar</Text>
         </TouchableOpacity>
+
         <TouchableOpacity style={styles.menu} onPress={this.onWeekPress.bind(this)}>
           <Text style={styles.menuText}>Week Calendar</Text>
         </TouchableOpacity>
```

Comment:
You forgot to add the button to open the timeline calendar screen to the menu

Reply:
Its Added Now!